edgexfoundry/app-functions-sdk-go
# refactor!: Rework code for refactored stand alone MessageBus Configuration

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-functions-sdk-go/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

      TBD

## Testing Instructions
Run non-secure EdgeX stack with Device Virtual
Build and run template app service with DEBUG logging
Verify service receives Event from MessagBus

## New Dependency Instructions (If applicable)
